Rubberduck.Parsing/Symbols/DeclarationSymbolsListener.cs

```diff
@@ -133,11 +133,10 @@ private IEnumerable<IAnnotation> FindAnnotations(int line)
         private void DeclareControlsAsMembers(IVBComponent form)
         {
             if (form.Controls == null) { return; }
-            var msFormsLib = _state.DeclarationFinder.FindProject("MSForms");
 
             foreach (var control in form.Controls)
             {
-                var typeName = control.TypeName();
+                var typeName = $"MSForms.{control.TypeName()}";
```

Comment:
You might want to make modifying the type name conditional ob the existence of the MSForms project. That would at least be better than nothing. 3rd party controls might still be a problem if MSForms is loaded.

Reply:
@MDoerner hmm good point, given [DeclareControlsAsMembers is called if the component has a designer](https://github.com/rubberduck-vba/Rubberduck/pull/3330/files/6a3a78d0d0cc5507452f3e8a8ef2a9f4f2f207a2#diff-222fda549176ed93cbe5cd0276ee044cR83); `MSForms` is guaranteed to be referenced if the `ComponentType` is a `ComponentType.UserForm`, but not if there's a designer... in which case I don't know what to qualify it with... but then yeah, in that case it probably doesn't even need to be qualified. I'll make the change.